### PR TITLE
Header

### PR DESCRIPTION
**Pull request by shsteimer**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--audi--aemsites.hlx.live/uk/web/en/used-cars
- After: https://header--audi--aemsites.hlx.live/uk/web/en/used-cars

### CODE REVIEW

File: aemedge/blocks/header/header.js [hunk 1]
```diff
@@ -1,148 +1,241 @@
...
+    // eslint-disable-next-line no-console
+    console.error('Failed to fetch audi nav json (likely due to CORS). Falling back to local.', e);
+  }
+
+  if (!navJson) {
+    const resp = await fetch(`${window.hlx.codeBasePath}/blocks/header/nav-fallback.json`);
+    if (resp.ok) {
+      navJson = await resp.json();
+    }
+  }
```
Comment (shsteimer) [inline aemedge/blocks/header/header.js:271]: @kunwarsaluja here's how I'm fetching header json with a local fallback, incase you want to do similar for footer.

**Review by kunwarsaluja — DISMISSED** [review]
Looks and works pretty good overall. Some tiny callouts if we want to do it now or in other issue #27 both works, See below screenshot

1. Header covers the full screen width in high viewpoints
**Live site**
Screenshot 2024-03-08 at 9 09 50 AM

**Edge Delivery**
Screenshot 2024-03-08 at 9 08 36 AM

2. in Mobile view, once i click a sub-menu like Models, there is transition at open and close of the menu underneath. Transition at open doesnt work but close does.

3. Header is Audi live site isn't sticky when scrolled so we need to remove the styling around it.

**Review by kunwarsaluja — CHANGES_REQUESTED** [review]
See comments

**Comment by shsteimer** [comment]
@kunwarsaluja all 3 things should now be resolved

**Review by kunwarsaluja — APPROVED** [review]

**Pull request merged**